scripts/unpack

```diff
@@ -27,11 +27,11 @@ fi
 
 STAMP="${PKG_BUILD}/.libreelec-unpack"
 
-mkdir -p ${BUILD}
+mkdir -p ${BUILD}/build
 
 # Perform a wildcard match on the package to ensure old versions are cleaned too
 PKG_DEEPHASH=
-for i in ${BUILD}/${PKG_NAME}-*; do
+for i in ${BUILD}/${PKG_NAME}-* ${BUILD}/build/${PKG_NAME}-*; do
```

Comment:
Same as with `scripts/clean`, don't bother supporting legacy build directories - this just needs to be:
```
for i in ${BUILD}/build/${PKG_NAME}-*; do
```